project_app/views.py

```diff
@@ -34,4 +33,26 @@ class Home(View):
     def get(self, request):
         return render(request, "homePage.html")
 
-# class AddAccount(View):
+
+class AccountManagement(View):
+    def get(self, request):
+        emails = list(map(UserModel.email))
+        return render(request, "accountManagement.html", {"emails": emails})
+
+
+class AddAccount(View):
+    def get(self, request):
+        return render(request, "addAccountPage.html")
+
+    def post(self, request):
+        driver = Driver()
```

Comment:
driver = request.session["driver"] # to fetch the driver rather than init a new one